regulations/static/regulations/js/source/views/main/preamble-view.js

```diff
@@ -33,13 +42,33 @@ var PreambleView = ChildView.extend({
     this.$read.show();
   },
 
+  handleParagraphActive: function(id) {
+    // update current Section ID as active paragraph changes
+    this.currentSectionId = id;
+  },
+
   handleWrite: function(e) {
-    var $target = $(e.target);
-    this.write(
-      $target.data('section'),
-      $target.data('label'),
-      $target.closest('[data-permalink-section]')
-    );
+    // + Write a comment about section (link)
+    if (e) {
```

Comment:
I broke it out into two separate functions again. I think the new function names are descriptive enough.